contributor/03-contributor-ethos-article.asciidoc

```diff
@@ -14,130 +14,125 @@ behave in a certain way:
-exercises in the [Open Organization
-Workbook](https://opensource.com/open-organization/resources/workbook).
+exercises in the https://opensource.com/open-organization/resources/workbook[Open Organization
+Workbook].
 
 Now, how do you figure out where an InnerSource project would like to discuss
-changes, future direction of the project? Many InnerSource projects outline how
+changes and future direction of the project? Many InnerSource projects outline how
 they like to be approached by potential Contributors in their README.md. If that
 document becomes too large to handle, contribution guidelines tend to be split
 out into a file named CONTRIBUTING.md files. Following those recommendations
-greatly helps the contributor selling their offer.
+greatly helps Contributors selling their offer.
 
 In all of these interactions, be prepared to "sell" your contribution to the
 host team. Articulate the value that the contribution will bring to their
 ecosystem.
 
 The host team will be the one taking over maintenance for your changes. It makes
-sense to offer to fulfill a [30-day
-warranty](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/30-day-warranty.md)
+sense to offer to fulfill a https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/30-day-warranty.md[30-day
+warranty]
 on your submission. This can
-alleviate the host team's fear of the contributors not being available for
+alleviate the host team's fear of the Contributors not being available for
 support with fixing bugs after the time on contribution.
 
-## Anticipate and follow house rules
+=== Anticipate and follow house rules
 
-When visiting your neighbors, they will likely help you around in their
-apartment: they'll show you the way to their living room and where the restroom 
+When you are visiting your neighbors, they will likely help you around in their
+apartment: they'll show you the way to their living room and where the restroom
 is located. If you're staying longer, they will probably
-give you more details: in my case an example would be to try to avoid turning on
+give you more details: in my case an example would be to avoid turning on
 the dishwasher and the electric kettle at the same time to avoid blowing the
 fuse.
 
-Similarly every software system comes with its own quirks and intricacies.
+Similarly, every software system comes with its own quirks and intricacies.
 Often the most obvious ones are well documented. In smaller projects this
 documentation can be found in the README.md. In larger ones, contribution
 specific documentation can often be found in the CONTRIBUTING.md document.
 
-Often what you'll find in this documentation is more information on how to
-checkout and build the project, how to run the test suite, how to submit changes
-to the project. Likely it will point you to further documentation if there it
+In these files you can expect to find information on how to
```

Comment:
These look like fine changes, but just wanted to make these grammatical changes along with the ASCIIDOC conversion?

Reply:
These come from @Ludmila-N's review. 
Ludmila's review was based on the Markdown version. To prevent really ugly merges and double conversions I've merged these in before the conversion, i.e. into this PR.

Reply:
Fantastic.